packages/firebase_app_check/firebase_app_check/android/src/main/java/io/flutter/plugins/firebase/appcheck/FlutterFirebaseAppCheckPlugin.java

```diff
@@ -33,6 +34,11 @@
 
   private static final String METHOD_CHANNEL_NAME = "plugins.flutter.io/firebase_app_check";
   private final Map<EventChannel, TokenChannelStreamHandler> streamHandlers = new HashMap<>();
+  private final String TAG = "FLTAppCheckPlugin";
+
+  private final String debugProvider = "debug";
+  private final String safetyNetProvider = "safetyNet";
```

Comment:
If it's deprecated, shouldn't we just remove it from this release? If a developer still wants to use it, they'll keep using the previous version?
So we can keep the previous API (debug or not) and just make breaking the type of provider?

Reply:
Yeah, that might be an idea. Although, it's not removed completely for another 2 years though so I feel users might complain if they get stuck on an old version 🤔. I can go either way but might be worth flagging with the team.  
<img width="924" alt="Screenshot 2022-10-06 at 09 11 23" src="https://user-images.githubusercontent.com/16018629/194258363-239830f9-703a-4ea9-8c8c-d29921bd7a3c.png">

Reply:
Would be great to have this. Current recommandations from firebase communication team is to ship new apps with this and safety net is marked as deprecated. Doesn't make sense to not have it. If app check is available on flutter we must have PlayIntegrity.   Well TBH i'll launch a brand new app soon and I'd like to have it. :D